README.md

```diff
@@ -143,22 +143,21 @@ Since the resource state is stored outside of the container, the GPU cluster lif
 ## Instructions
 
 1. gcloud auth application-default login.
```

Comment:
Sometime I've seen that you need to use both `gcloud auth login` and `gcloud auth application-default login`. There is command that combines both `gcloud auth login --update-adc`. Should we use that instead ?